### ISSUE TITLE
Accessibility: Screen readers don't work with Atom at all

### ISSUE THREAD

**Issue by zersiax** [atom/atom#18660]
### Prerequisites

* [x ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Screen readers appear to have no access at all to the contents of the editor, making the entire editor useless for screen reader users. For "an editor of the 21st century" that lets me "make it my own editor" I would expect accessibility to be excellent, added from the design stage and improved regularly. Instead, I find a complete lack of accessibility for screen reader users.

### Steps to Reproduce

1. With a screen reader like NVDA or NArrator on Windows or VoiceOver on Mac OS X running, open Atom.
2. Type text and try to read it back using up and down arrow keys. 

**Expected behavior:** [What you expect to happen]
Screen readers are able to detect the text in the editor.
**Actual behavior:** [What actually happens]
Either deafening silence or the word "blank" being uttered, meaning there's no text on the given line.

**Reproduces how often:** [What percentage of the time does it reproduce?]
100%
### Versions
Any Atom version
### Additional Information
Vs Code initially had this issue, but they have overcome it. You might be able to reproduce some of their efforts.

**Comment by 50Wliu** [comment]
Thanks for taking the time to contribute!

We noticed that this is a duplicate of #1691. You may want to subscribe there for updates.

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).
